gulpfile.js

```diff
@@ -60,5 +60,5 @@ gulp.task('scss-unused-variables', getTask('scss-unused-variables'));
 gulp.task('ci', ['scss-lint', 'scss-unused-variables']);
 
 gulp.task('build', function (done) {
-    runSequence('clean:dist', 'sass:build', 'svgs-generate', 'jekyll:docs', 'docs:copy-components', 'fingerprint', 'fingerprint-replace', 'sass:docs-build', done);
+    runSequence('clean:dist', 'sass:build', 'sass:docs-build', 'svgs-generate', 'jekyll:docs', 'docs:copy-components', 'fingerprint', 'fingerprint-replace', done);
```

Comment:
We need to fingerprint docs css as well so it needs to be done before fingerpinting tasks